pynicotine/gtkgui/ui/settings/search.ui

```diff
@@ -227,9 +273,26 @@
                 <property name="visible">1</property>
                 <property name="focusable">0</property>
                 <child>
-                  <object class="GtkEntry" id="FilterOut">
+                  <object class="GtkBox">
                     <property name="visible">1</property>
-                    <property name="tooltip-text" translatable="yes">Filter out results whose file paths contain the specified text. Multiple phrases and words can be specified, e.g. exact phrase|music|term|exact phrase two</property>
+                    <property name="focusable">0</property>
+                    <child>
+                      <object class="GtkLabel">
+                        <property name="visible">1</property>
+                        <property name="label">            </property>
```

Comment:
I would be glad of any suggestion how to do an indent, because I don't like these blank labels?